src/core/solvers/kkt.cpp

```diff
@@ -257,13 +257,14 @@ void SolverKKT::allocateData() {
   nu_ = 0;
   const std::size_t& nx = problem_->get_nx();
   const std::size_t& ndx = problem_->get_ndx();
-  const std::size_t& nu = problem_->get_nu_max();
+  //const std::size_t& nu = problem_->get_nu_max();
   for (std::size_t t = 0; t < T; ++t) {
     if (t == 0) {
       xs_try_[t] = problem_->get_x0();
     } else {
       xs_try_[t] = Eigen::VectorXd::Constant(nx, NAN);
     }
+    const std::size_t& nu = problem_->get_runningModels()[t]->get_nu();
```

Comment:
I didn't get this.
Why do you change from `nu_max` to `nu`?

Reply:
`kkt` solver doesn't have a notion of nu_max. This should have always been `sum(nu[i])`

Reply:
Ok, I see. Thanks for the clarification :)